.github/workflows/build-test-publish.yml

```diff
@@ -125,7 +125,7 @@ jobs:
           ctest -j 2 -VV -S dashboard.cmake
         shell: cmd
 
-  build-test-superbuild:
+  build-test-python:
```

Comment:
If these checks are intended to cover both superbuild and Python example functionality, could we use a name indicative of that intention such as `superbuild-test-python`?

Reply:
Yes, we could use `python-superbuild`.